Userland/Libraries/LibGL/GLVert.cpp

```diff
@@ -165,6 +175,21 @@ void glTexCoord2i(GLint s, GLint t)
     g_gl_context->gl_tex_coord(s, t, 0.0f, 1.0f);
 }
 
+void glTexCoord3f(GLfloat s, GLfloat t, GLfloat r)
+{
+    g_gl_context->gl_tex_coord(s, t, r, 1.0f);
+}
+
+void glTexCoord3fv(GLfloat const* v)
+{
+    g_gl_context->gl_tex_coord(v[0], v[1], v[2], 1.0f);
+}
+
+void glTexCoord4f(GLfloat s, GLfloat t, GLfloat r, GLfloat q)
+{
+    g_gl_context->gl_tex_coord(s, t, r, q);
+}
```

Comment:
Any specific reason for not including the `fv` version of `glTexCoord4`?

Reply:
It's already there :^)
https://github.com/SerenityOS/serenity/blob/10a52c0b3214eddfb053f4e10d1412b79b1bfeb1/Userland/Libraries/LibGL/GLVert.cpp#L168-L171

Reply:
whoops, missed that